web3/pm.py

```diff
@@ -71,6 +79,9 @@
 # >>> w3.pm
 # <web3.pm.PM at 0x....>
 
+ReleaseData = namedtuple("ReleaseData", ["package_name", "version", "manifest_uri"])
```

Comment:
Unless you have to support Python 3.5 there's a cleaner syntax for `NamedTuple`.

```python
class ReleaseData(NamedTuple):
    package_name: str
    version: str
    manifest_uri: str
```

Reply:
Nice, thanks for the tip. I vaguely remember trying to use this syntax before in `3.6` and it resulted in some obscure error that wasn't fixed until `3.7` - maybe had something to do with [setting defaults](https://github.com/python/mypy/issues/4788) :shrug: but everything seems to be working fine here.